test/app_page_scraper_test.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 import pytest
-from src import PlayScrapingError, PlayAppPageScraper, AccessError
+from src import PlayScrapingError, PlayAppPageScraper, AccessError, SCRAPER_INFO_FILE_NAME
 
 ICON_HTML = """
```

Comment:
If you don't want these to be imported anywhere, I would suggest prepending them with a _